docker/Dockerfile

```diff
@@ -26,13 +26,18 @@ USER opam
 WORKDIR /home/opam
 RUN opam init --disable-sandboxing --auto-setup --bare
 
-# Copy opam file for dependency resolution then install dependencies
-COPY --chown=opam:opam *.opam ./
+# Root handles file copying and directory setup
+USER root
+COPY --chown=opam:opam *.opam /home/opam/
+RUN mkdir -p /home/opam/geneweb && chown opam:opam /home/opam/geneweb
+COPY --chown=opam:opam . /home/opam/geneweb/
+
```

Comment:
I don't understand why you must be root there. These commands should work:
```dockerfile 
RUN mkdir /home/opam/geneweb 
COPY --chown=opam:opam . /home/opam/geneweb
``` 
`/home/opam` is not owned by `opam` user?